packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -197,6 +198,10 @@ storiesOf('TooltipTrigger', module)
         </Flex>
       </Flex>
     )
+  )
+  .add(
```

Comment:
even better, lets add a test for this
I am doing that for ActionMenu right now as well

Reply:
will do